handbook/company/communications.md

```diff
@@ -213,31 +213,21 @@ Fleet prefixes all GitHub labels with special characters or words to organize an
 
-Opinionated conventions help people work faster and spend less time figuring out what to name things, or misunderstanding why they're named what they are.  This also reduces the total number of labels required while maintaining an expressive labeling system. For example, instead of a label called `platform-dev-backend`, we use `#platform :dev ~backend`.
+Opinionated conventions help people work faster and spend less time figuring out what to name things, or misunderstanding why they're named what they are.  This also reduces the total number of labels required while maintaining an expressive labeling system.
 
 Labels with a `#g-` prefix refer to a kanban board. Since it is best practice to have an issue on a single board, make an effort to have only one label with the `#g-` prefix per issue.
 
 > _**Note:** There are only a few "special" labels that are exceptions to this rule:
-> - `bug`
-> - `story`
-> -  `prospect-` _(TODO: This makes sense, but I noticed on Nov 30, 2023 that `customer-` was being used in at least one place `prospect-` was supposed to be.  Let's decide what we'll be doing.  Easiest thing is probably to switch to `customer-` for everything and then clarify who is a customer in the mapping between labels and actual organization names.  Up to Luke and Zay.)_
-> - `customer request` _(FUTURE: is this needed?  if so, could use special symbol prefix in front of it)_
-> -  `needs prioritization` _(FUTURE: is this needed? could we use a symbol prefix in front of it?)_
-> - `github_actions` _(TODO: why is this here?  Instead we can use prefixes, like `~`)_
-> - `docker` _(TODO: why is this here?  Instead we can use prefixes, like `~`)_
-> -  `go` _(TODO: why is this here?  Instead we can use prefixes, like `~`)_
-> -  `javascript` _(TODO: why is this here?  Instead we can use prefixes, like `~`)_
-> - `Epic` _(TODO: Find a way to remove this.  It is an artifact from Zenhub and not something we actually want to exist or use, as it is confusing.)_
-> -  `p4` _(TODO: why is this here?  Instead we can use prefixes, like `~`.  Or better yet, delete it.  What is a P4 anyway?)_
-> -  `p5` _(TODO: why is this here?  Instead we can use prefixes, like `~`.  Same as `p4`.)_
-> -  `story to demo` _(FUTURE: is this needed?  if so, could use special symbol prefix in front of it)_
-> -  `bug to demo` _(FUTURE: is this needed?  if so, could use special symbol prefix in front of it)_
+> - `bug` A defect in the product.
+> - `bug-` Defect category. For example, `bug-enrollment`, `bug-profiles-sync`, `bug-policies`. This allows us to track the areas of the product producing the most bugs.
+> - `story` A user story.
+> -  `prospect-` A customer prospect.
+> - `Epic` Do not use. _(TODO: ZenHub automatically recreates this label when we group sub-tasks inside of a user story. Find a way to remove this. It is an artifact from Zenhub and not something we actually want to exist or use, as it is confusing.)_
```

Comment:
I'm leaving this here until we can figure out a way to delete it permanently. We've asked ZenHub, and they said they have no way to disable it and don't plan to make that a feature. 

Because ZenHub automatically creates this label if it's not present, we would have to update the Fleet GitHub bot to watch for this label and delete it any time it's created. I'd expect that to happen a few times a week, at least. 

Alternatively, we could escalate our feature request to ZenHub.